src/views/simplified-account/settings/card-payments/collect-billing-address.njk

```diff
@@ -0,0 +1,45 @@
+{% extends "../settings-layout.njk" %}
+
+{% block settingsPageTitle %}
+  Card payments
+{% endblock %}
+
+{% block settingsContent %}
+  {% if isReadOnly %}
+    {% include "includes/settings-read-only.njk" %}
+  {% endif %}
+  <form method="post">
+    <input id="csrf" name="csrfToken" type="hidden" value="{{csrf}}"/>
+    {{ govukRadios({
+      name: "collectBillingAddress",
+      fieldset: {
+        legend: {
+          text: "Collect billing address",
+          classes: "govuk-fieldset__legend--l",
+          isPageHeading: true
+        }
+      },
+      hint: {
+        text: 'Your payment service provider (PSP) uses the billing address to carry out fraud checks. If you turn off billing address collection, your PSP or your users\' banks may decline more payments.'
+      },
```

Comment:
This differs from the content in the designs - is this a revised version of the content provided by the design team?

Reply:
Yes, we had some queries and reviewed with Alan and Sally.